scripts/makebumpver

```diff
@@ -206,9 +214,15 @@ class MakeBumpVer:
         proc = run_program(['git', 'config', field])
         return proc[0].strip('\n')
 
-    def _incrementVersion(self):
+    def _incrementVersion(self, bump_major_version=False, add_version_number=False):
         fields = self.version.split('.')
-        fields[-1] = str(int(fields[-1]) + 1)
+        if add_version_number:  # add another version number to the end
+            fields.append(str(int(1)))
+        elif bump_major_version:  # increment major version
+            fields[0] = str(int(fields[0]) + 1) # bump major version
+            fields[-1] = str(int(1)) # reset minor version to 1
+        else:  # minor version bump
+            fields[-1] = str(int(fields[-1]) + 1)
```

Comment:
Could you please change these numbers to constants.

Reply:
Sure. :)